revolv/templates/project/partials/payment_modal.js

```diff
@@ -378,6 +389,13 @@ $('#donate-form').submit(function(e) {
         $donateModalErrors.children('.error-msg')
             .text('Your card information is invalid. Please correct it.');
         $('#donate-modal').foundation('reveal', 'open');
+    }).always(function () {
+        paymentSpinner.stop();
+        setTimeout(function() {
```

Comment:
This deserves a comment saying why we need to do setTimeout (i.e. that if we don't the modal fading animation overlaps)